### PR TITLE
Changes in models, performed migration

### PR DESCRIPTION
**Pull request by livike**
Changes in models, 
performed migration,
updates requirements.txt

ref. issue #105

### CODE REVIEW

File: api/admin.py [hunk 1]
```diff
@@ -1,3 +1,8 @@
 from django.contrib import admin
+from .models import Task
```
Comment (goranche) [inline api/admin.py]: could this be simplified?

    from .models import *

**Comment by goranche** [comment]
why are there 3 migrations already?

File: api/admin.py (hunk 1 above)
Comment (livike) [inline api/admin.py]: Done

**Comment by livike** [comment]
3 migrations because I made some improvements on models

**Comment by goranche** [comment]
given that we don't have a database yet, you can delete all the migration and recreate the initial one

**Pull request merged**